web-client/iron-remote-desktop/src/iron-remote-desktop.svelte

```diff
@@ -51,8 +55,7 @@
 
     let viewerStyle = $state('');
     let wrapperStyle = $state('');
-
-    let wasmService = new WasmBridgeService();
+    let wasmService = new WasmBridgeService(module);
```

Comment:
thought: `WasmBridgeService` should probably be renamed since we are not wrapping a WASM necessarily anymore. `RemoteDesktopService` may be appropriate? What do you think?

Reply:
Renamed to `RemoteDesktopService`